This patch fixes the name of the wheels and the package from onnxruntime-gpu to onnxruntime.
Author: Pavel Tomanek (Inuits)
--- setup.py.orig	2025-12-16 13:25:52.996185000 +0100
+++ setup.py	2025-12-16 13:26:05.009368000 +0100
@@ -797,7 +797,7 @@
         elif qnn_version:
             f.write(f"qnn_version = '{qnn_version}'\n")
 
-
+package_name =  "onnxruntime"
 save_build_and_package_info(package_name, version_number, cuda_version, rocm_version, qnn_version)
 
 extras_require = {}
